tests/Zend/Ldap/.ci/basedn.ldif

```diff
@@ -0,0 +1,3 @@
+dn: ou=test,dc=example,dc=com
```

Comment:
please keep consistency between files and if possible add empty line at the end of file

Reply:
both ldif files not do not an empty line at the end, do you mean we should add empty line to both?

Reply:
no let's not add empty lines here. those are special files.

Reply:
not empty line but line terminator